rasa/core/channels/telegram.py

```diff
@@ -202,11 +209,12 @@ async def set_webhook(_: Request) -> HTTPResponse:
         async def message(request: Request) -> Any:
             if request.method == "POST":
 
-                if not out_channel.get_me()["username"] == self.verify:
+                request_dict = request.json
+                update = Update.de_json(request_dict)
```

Comment:
what happened to the `out_channel` argument?

Reply:
Not sure how this worked before but `get_me` returns a user object.